packages/drivers/odsp-driver-definitions/src/tokenFetch.ts

```diff
@@ -83,6 +99,29 @@ export const tokenFromResponse = (
 		? null
 		: tokenResponse.token;
 
+/**
+ * Helper method which transforms return value for TokenFetcher method to Authorization header value
+ * @param tokenResponse - return value for TokenFetcher method
+ * @returns Authorization header value
+ * @internal
+ */
+export const authHeaderFromTokenResponse = (
+	tokenResponse: string | TokenResponse | null | undefined,
+): string | null => {
+	if (
+		tokenResponse !== null &&
+		typeof tokenResponse === "object" &&
+		tokenResponse.authorizationHeader !== undefined
```

Comment:
You can simplify it - tokenResponse?.authorizationHeader is sufficient. Everything is an object in JS

Reply:
I can't do that because tokenResponse could be a string. But I can get rid of the null check because the typeof check will cover it.